--- rc.sysinit	2012-05-10 17:29:23.795096265 +0000
+++ rc.sysinit.chkusrlocal	2012-05-10 17:29:10.089095008 +0000
@@ -204,6 +204,15 @@
     init_crypto 0
 fi
 
+# TJL 2009/10/07: The mkvm script creates /dev/hdb but can't make
+# partitions or filesystems from outside the guest OS -- this
+# chkusrlocal script sees if /dev/hdb has a partition and makes one if
+# not, creating a filesystem on it and adding it as /usr/local to
+# /etc/fstab
+if [ -f /sbin/chkusrlocal ]; then
+	/sbin/chkusrlocal
+fi
+
 if [ -f /fastboot ] || strstr "$cmdline" fastboot ; then
 	fastboot=yes
 fi
